data/grc/Arnott_1996_Athenaeus/stemma.gv

```diff
@@ -0,0 +1,23 @@
+# Athenaeus: Arnott 1996
+# Epitome derived from A, contamination from a sibling of A
+
+digraph {
+
+    omega -> A;
+    omega -> sibling_of_A;
+    A -> epitome;
+    sibling_of_A -> epitome [style="dashed"]; # hypothetical ms. derived from the archetype, by definition
+    epitome -> gamma;
+    gamma -> delta;
+    delta -> C;
+    delta -> E;
+    gamma -> Eustathius_ms;
+
+    omega [color="grey", label="ω"]; # Arnott names the archetype α
+    sibling_of_A [color="grey", label=""];
+    epitome [color="grey", label="Epitome"];
+    Eustathius_ms [color="grey", label="Eustathius’s\nmanuscript"];
```

Comment:
`MS` (and no linebreak ?)

Reply:
I went for a consistent name for this manuscript across the `stemma.gv` files. It was partly related to my trying to make the output look good, which is not a priority. But does it make sense to respect the editor's "MS"? I am not sure, unless we aim at "diplomatic" stemmata. There are clearly conflicting usages in naming manuscripts and abbreviating the word "manuscript", as well as stemmata sources written in different languages. Do you have a preference?

Reply:
Non, I let you decide the way you think more relevant for harmonising ! You're the expert on this ;)